### DISCUSSION TITLE
Windows 10 1903 Start Menu

### DISCUSSION THREAD

**Discussion by XuanChen1234** [valinet/ExplorerPatcher#2254] [Ideas]
Is it possible to Windows 10 1903 start menu into ExplorerPatcher?
![images (5)](https://github.com/valinet/ExplorerPatcher/assets/140480110/12f20428-3887-4c74-92e7-e8aa74eda851)

**Comment by Antonomasia3rd** [comment]
iirc the key needed to make it non-transparent (i found this on 20H2, as well as reddit), is removed on Windows 11
`Computer\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FeatureManagement\Overrides\0\2093230218`

Even manually adding a New Key and DWORDs don't help
![image](https://github.com/valinet/ExplorerPatcher/assets/89201774/cf00b28f-12ad-4b05-be8e-2c9c458089ee)

**Reply by Tourniquet88** [reply]
This has been discussed here (https://github.com/valinet/ExplorerPatcher/discussions/1729) already. Seems like the code is still present.

**Reply by Antonomasia3rd** [reply]
ty for informing ^^
